src/framework/uicomponents/view/menuitem.h

```diff
@@ -149,6 +154,8 @@ public slots:
     QList<MenuItem*> m_subitems;
 
     ui::UiAction m_action;
+
+    QString m_nativeMenuBarIconPath = "";
```

Comment:
```suggestion
    QString m_nativeMenuBarIconPath;
```
(`""` is not the same as `QString()`; the latter is more efficient.)